main/lsp/wrapper.cc

```diff
@@ -46,25 +27,11 @@ vector<unique_ptr<LSPMessage>> LSPWrapper::getLSPResponsesFor(unique_ptr<LSPMess
 }
 
 vector<unique_ptr<LSPMessage>> LSPWrapper::getLSPResponsesFor(vector<unique_ptr<LSPMessage>> &messages) {
-    // Determine boolean before moving messages.
-    bool foundPostInitializationRequest = !messages.empty();
-    if (!initialized) {
-        foundPostInitializationRequest = false;
-        for (auto &message : messages) {
-            if (initialized) {
```

Comment:
LSPConfiguration is now the single source-of-truth of initialization, so no need for this duplicate logic!